demisto_sdk/commands/lint/helpers.py

```diff
@@ -297,14 +297,15 @@ def get_python_version_from_image(image: str, timeout: int = 60, log_prompt: str
     if 'pwsh' in image or 'powershell' in image:
         return '3.8'
 
-    docker_user = os.getenv('DOCKERHUB_USER')
-    docker_pass = os.getenv('DOCKERHUB_PASSWORD')
+    match_group = re.match(r'[\d\w]+/python3?:(?P<python_version>[23]\.\d+)', image)
+    if match_group:
+        return match_group.groupdict()['python_version']
+
     docker_client = docker.from_env(timeout=timeout)
-    docker_client.login(username=docker_user,
-                        password=docker_pass,
-                        registry="https://index.docker.io/v1")
```

Comment:
I think that this should stay here (there is a pool limit for docker environments that are logged in [[see here](https://developers.redhat.com/blog/2021/02/18/how-to-work-around-dockers-new-download-rate-limit-on-red-hat-openshift#docker_s_new_rate_limit)]).
any way can (and should move it to the global scope (there no reason to create the client and login for each command running).